core-web/libs/sdk/client/src/lib/editor/models/client.model.ts

```diff
@@ -7,6 +7,7 @@ declare global {
 
 export const INITIAL_DOT_UVE: DotUVE = {
     editContentlet,
+    reorderMenu,
     lastScrollYPosition: 0
 };
```

Comment:
Careful about just throwing things to the window without any control because this create another way of communication with the UVE and it will be get out of control very fast.